astropy/constants/__init__.py

```diff
@@ -26,9 +26,9 @@
 from .constant import Constant, EMConstant  # noqa
 from . import si  # noqa
 from . import cgs  # noqa
-from .config import codata, iaudata
+from .config import codata, iaudata  # noqa
 
-from . import utils as _utils
+from . import utils as _utils  # noqa
```

Comment:
It seems to me this line can just be deleted, since a local import is done below.

Reply:
Actually, I don't think so, due to this line outside the function:

https://github.com/astropy/astropy/blob/86ec85aa9a5881c1ec8cf67c22aebb870f686e2b/astropy/constants/__init__.py#L44

And because we `del _unit` at the bottom of the module, we have to re-import it inside the function as well. Please let me know if I got it wrong.

Reply:
OK, you're right. Sorry for the noise.